pkg/controller/kibana/network/ports.go

```diff
@@ -0,0 +1,10 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package network
+
+const (
+	// HTTPPort is the (default) port used by Kibana
+	HTTPPort = 5601
```

Comment:
why did you move this to its own sub-package?

Reply:
To reuse it without import cycle. Same for the kbnamer. Another option could be to move it in `pkg/apis/kibana/v1` but I replicated what we did for ES.